index.js

```diff
@@ -104,9 +94,9 @@ routerSwitch();
  */
 
 //For testing purposes. True=Logged in.
```

Comment:
Maybe move code that is just for testing to a separate file and just import it in ?

Reply:
I've no idea who added that, but yes, I agree.

Reply:
Can you move it ?

Reply:
I moved the functions and constants to a testscript folder, fixed spelling error, configured for export, but they are not getting called anywhere, seeing as they were already commented out.